scalableminds/webknossos
# Compress Volume Buckets Already in Frontend

### URL of deployed dev instance (used for testing):
- https://lzfrontend.webknossos.xyz

### Steps to test:
- Do some volume tracing, save, reload, buckets should load correctly
- download + upload

### Issues:
- contributes to #4594 

------
